picard/ui/filter.py

```diff
@@ -70,6 +75,13 @@ def __init__(self, parent=None):
 
         self.initializing = False
 
+    def _get_saved_selected_filters(self):
+        config = get_config()
+        temp = config.persist[self.saved_filters_key]
+        if temp is not None:
+            return set(temp)
```

Comment:
We shouldn't load those blindly from config file in case those changed, but rather compare to filterable tags and include only those in config that are also available in running instance.

So basically it should be the intersection of both sets.